vaadin-testbench-unit/src/test/java/com/vaadin/flow/component/tabs/TabsView.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2022 Vaadin Ltd
+ *
+ * This program is available under Commercial Vaadin Developer License
+ * 4.0 (CVDLv4).
+ *
+ *
+ * For the full License, see <https://vaadin.com/license/cvdl-4.0>.
+ */
+
+package com.vaadin.flow.component.tabs;
+
+import com.vaadin.flow.component.Component;
+import com.vaadin.flow.component.HasComponents;
+import com.vaadin.flow.component.Tag;
+import com.vaadin.flow.router.Route;
+
+@Tag("div")
+@Route(value = "tabs", registerAtStartup = false)
+public class TabsView extends Component implements HasComponents {
+
+    Tabs tabs;
+    Tab details;
+    Tab payment;
+    Tab shipping;
+
+    public TabsView() {
+        details = new Tab("Details");
+        payment = new Tab("Payment");
+        shipping = new Tab("Shipping");
```

Comment:
The tab with components can basically contain anything so `select(String)` will at least not work with it. `select(index)` should work fine and could be tested, but one question is that could/should we have a `find(ComponentQuery)` that would be used to run the query for tabs to find the correct one.

This could be a generic one to be used for other components also may that contain components in the "items"

Reply:
It seems a nice addition to have, but maybe we can collect similar use cases in other wrappers and implement the feature it in a separate PR

Reply:
Added issue #1455